scio-avro/src/main/scala/com/spotify/scio/avro/types/SchemaUtil.scala

```diff
@@ -29,11 +26,12 @@ import scala.collection.JavaConverters._
 object SchemaUtil {
 
   /** Convert schema to case class definitions. */
-  def toPrettyString(schema: Schema, indent: Int = 0): String =
-    getCaseClass(schema.getFields, schema.getName, indent)
+  def toPrettyString(className: String, schema: Schema, indent: Int = 0): String =
```

Comment:
use `schema.getName` as `className` so you can keep the function signature?

Reply:
If I would do that, than the prettyString wouldn't match the class hierarchy exactly.